pixi.toml

```diff
@@ -0,0 +1,138 @@
+[project]
+name = "param"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+[tasks]
+install = 'python -m pip install --no-deps --disable-pip-version-check -e .'
+
+[activation.env]
+PYTHONIOENCODING = "utf-8"
+
+[environments]
+test-38 = ["py38", "test-core", "test", "example", "test-example"]
+test-39 = ["py39", "test-core", "test", "example", "test-example"]
+test-310 = ["py310", "test-core", "test", "example", "test-example"]
+test-311 = ["py311", "test-core", "test", "example", "test-example"]
+test-312 = ["py312", "test-core", "test", "example", "test-example"]
+test-313 = ["py313", "test-core", "test", "example", "test-example"]
+test-core = ["py313", "test-core"]
+test-pypy = ["pypy", "test-core", "test-pypy"]
+docs = ["py311", "example", "doc"]
+build = ["py311", "build"]
+lint = ["py311", "lint"]
+
+[dependencies]
+nomkl = "*"
+pip = "*"
+
+[feature.py38.dependencies]
+python = "3.8.*"
+
+[feature.py39.dependencies]
+python = "3.9.*"
+
+[feature.py310.dependencies]
+python = "3.10.*"
+
+[feature.py311.dependencies]
+python = "3.11.*"
+
+[feature.py312.dependencies]
+python = "3.12.*"
+
+[feature.py313.dependencies]
+python = "3.13.*"
+
+[feature.pypy]
+platforms = ["linux-64", "osx-64", "win-64"] # not supported for osx-arm64
+
+[feature.pypy.dependencies]
+pypy = "7.3.*"
+
+[feature.example.dependencies]
+aiohttp = "*"
+pandas = "*"
+panel = "*"
+
+# =============================================
+# =================== TESTS ===================
+# =============================================
+[feature.test-core.dependencies]
+pytest = "*"
+pytest-asyncio = "*"
+pytest-cov = "*"
+pytest-github-actions-annotate-failures = "*"
```

Comment:
Not a big fan of this plugin. It's also definitely not part of the `core` deps needed to tun the tests.

Reply:
> I'm a big fan of it. Reduces the need to look into the logs.

My experience of it so far has been to give me the impression that, basically, every Panel PR is failing as it also displays errors that occur during flaky tests.

---

> The same could be set about pytest-cov. The plugin is a no-op if you are not in a Github action and only have pytest as a dependency.

Reminds me of the discussion we recently had about `hatch-conda-build ` :D `pytest-cov` can be used by a user, I don't think `pytest-github-actions-annotate-failures` can?

But anyway, I'm again fine with how this is.

Reply:
> The correct thing to be frustrated about is the flaky test themselves 🙂

Both things are frustrating.